test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/RequestTests.cs

```diff
@@ -115,6 +115,21 @@ public class RequestTests
         }
 
         [ConditionalFact]
+        public async Task Request_FullUriInRequestLine_ParsesPath()
+        {
+            string root;
+            using (var server = Utilities.CreateHttpServerReturnRoot("/", out root))
+            {
+                // Send a HTTP request with the request line:
+                // GET http://localhost:5001 HTTP/1.1
```

Comment:
Assert `root` does not have a trailing slash. Utilities may be refactored later and unintentionally break this.